src/Core/Commander/Providers/Fetcher.js

```diff
@@ -2,6 +2,8 @@ import { TextureLoader } from 'three';
 
 const textureLoader = new TextureLoader();
 
+textureLoader.setCrossOrigin('');
```

Comment:
> I guess we probably want with-credentials? Just in case there is a http auth on the wms/wmts/whatever server...

Maybe this should be configurable per source?

In any case, the XHR wasn't previously done with `withCredentials=true` (in iTowns), so if this change is made, it should be separate from the Three.js update from this PR.